app/controllers/shipit/shipit_controller.rb

```diff
@@ -32,9 +32,9 @@ def ensure_required_settings
 
     def force_github_authentication
       if current_user.logged_in?
-        team = Shipit.github_team
-        if team && !current_user.in?(team.members)
-          render text: "You must be a member of #{team.handle} to access this application.", status: :forbidden
+        teams = Shipit.github_teams
+        if teams && teams.none? { |t| current_user.in?(t.members) }
+          render text: "You must be a member of an authorized team to access this application.", status: :forbidden
```

Comment:
I didn't feel the need to expose internal configuration, but if you prefer the list of authorized teams, I'll re-add it.